src/caffe/proto/caffe.proto

```diff
@@ -393,8 +393,10 @@ message LayerParameter {
   optional ReductionParameter reduction_param = 136;
   optional ReLUParameter relu_param = 123;
   optional ReshapeParameter reshape_param = 133;
+  optional ROIPoolingParameter roi_pooling_param = 146;
```

Comment:
@Austriker Thanks for the faster_rcnn_end2end/tain.prototxt.Can you give me more information such as what is the type of the train set. It is better to give some examples.